package.json

```diff
@@ -22,7 +22,11 @@
   "author": "Angular2-Meteor (https://github.com/Urigo/angular2-meteor)",
   "license": "MIT",
   "scripts": {
-    "build": "gulp build",
+    "typings": "typings i",
+    "prebuild": "npm run typings && npm run lint",
+    "build": "npm run build-only",
+    "build-only": "tsc",
```

Comment:
@barbatus I was trying to fix them since it was a WIP or just to use `tsc || 0` if it wouldn't be possible.